SparkPost/2web2ui
# AC-652: Dashboard Role Changes

### What Changed
 - Dashboard available to other roles (except subaccount roles)
    - Checklist hidden from those missing grants instead of whole dashboard
    - Route prevents subaccount roles
    - Email users directed to dashboard, reporting still directed to summary report
 - Dashboard now shows usage from the start

### How To Test
 - Login to check where you are directed depending on Role
    - Admin, Developer, Email directed to dashboard
    - Reporting directed to summary w/ link to dashboard in navbar
    - Subaccount reporting directed to summary w/ no link to dashboard
 - Admin and developer should have access to checklist on dashboard

### To Do
- [ ] Double check the routes access 
- [ ] Address any UX concerns with new landing page
